sherlock/internal/deprecated_controllers/v2controllers/environment_test.go

```diff
@@ -273,6 +273,7 @@ func (suite *environmentControllerSuite) TestEnvironmentCreate() {
 			EditableEnvironment: EditableEnvironment{
 				DefaultCluster: &terraDevCluster.Name,
 				Owner:          testutils.PointerTo("some-email-not-in-the-database@broadinstitute.ogr"),
+				PactIdentifier: nil,
```

Comment:
```suggestion
```

Setting this field to nil is a no-op, it'll be nil if unset. To test your controller/api-level changes, I'd suggest adding to the edit tests in `environment_test.go` and `chart_test.go`; I think the methods will be called `TestEnvironmentEdit()` and `TestChartEdit()`.

Reply:
In ```testEnvironment```, was your goal to test edits in every environment and a suitable environment?

Reply:
1fff591